components/clock.rst

```diff
@@ -0,0 +1,95 @@
+.. index::
+   single: Clock
+   single: Components; Clock
+
+.. _`clock-component`:
+
+The Clock Component
+===================
+
+.. versionadded:: 6.2
+
+    The Clock component was introduced in Symfony 6.2
+
+The Clock component decouples applications from the system clock. This improves
+testability of time-sensitive logic.
```

Comment:
```suggestion
The Clock component decouples applications from the system clock. This allows
you to fix time to improve testability of time-sensitive logic.
```

just a suggestion, but I think we should clarify a bit on the *why* about improving testability